go/utils_test.go

```diff
@@ -238,8 +238,11 @@ func TestEscapeBytesBackslash(t *testing.T) {
 	r = escapeBytesBackslash([]byte{}, []byte{'\x1a'})
 	assert.Equal(t, string(r), "\\Z")
 
+	// Single quotes can be escaped by adding another single quote.
+	// https://docs.aws.amazon.com/athena/latest/ug/select.html
+	// https://docs.aws.amazon.com/athena/latest/ug/data-types.html#data-types-considerations
 	r = escapeBytesBackslash([]byte{}, []byte{'\''})
-	assert.Equal(t, string(r), `\'`)
+	assert.Equal(t, string(r), `''`)
```

Comment:
Is there a reason for changing this? Is backslash worse for some reason?

I guess this is just adding test coverage? 

Could we have this in addition instead of replacement?

Reply:
Per Athena's docs (https://docs.aws.amazon.com/athena/latest/ug/select.html#select-escaping), the way to escape a single quote is to prepend another single quote (`'` becomes `''`), and not by adding a backslash. I couldn't find any documentation on other characters, so I left the other special character handling the same.

Ah, the `#select-escaping` in the URL got cut off from the comment. I'll add that part in.